packages/api-gateway/src/drivers/http/routes/places/schema.js

```diff
@@ -20,6 +20,19 @@ const postSchema = {
   },
 };
 
+const deletePlaceSchema = {
+  description: 'This is the route to use for deleting a place',
+  tags: ['Places'],
+  body: {
```

Comment:
I think that is most common to pass the id in the url `params` for this kind of endpoints, can you update this to pass the id in the URL?
```suggestion
  params: {
```